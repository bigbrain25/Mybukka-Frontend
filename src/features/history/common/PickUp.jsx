import React from 'react';
import PropTypes from 'prop-types';

const PickUp = ({ title, pickupAddress, pickupContactMobile }) => (
  <div>
    <h5 className="text-uppercase font-size-14">{title}</h5>
    <h5 className="text-capitalize font-size-14">
      {pickupAddress}
    </h5>
    <h5 className="font-size-14">{pickupContactMobile}</h5>
  </div>
);

export default PickUp;

PickUp.propTypes = {
  title: PropTypes.string.isRequired,
  pickupAddress: PropTypes.string.isRequired,
  pickupContactMobile: PropTypes.string.isRequired
};
