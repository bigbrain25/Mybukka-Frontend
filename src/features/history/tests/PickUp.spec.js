import React from 'react';
import PickUp from '../common/PickUp';

describe('PickUp component', () => {
  const props = {
    title: 'title',
    pickupAddress: 'pickup address',
    pickupContactMobile: '0973738939',
  };

  const wrapper = mount(<PickUp {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
