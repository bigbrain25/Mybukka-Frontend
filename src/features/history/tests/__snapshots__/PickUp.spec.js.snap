// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PickUp component renders properly 1`] = `ReactWrapper {}`;
